JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/utils/BringIntoViewIfChildrenAreFocused.kt

```diff
@@ -17,39 +17,50 @@
 package com.google.jetstream.presentation.utils
 
 import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.foundation.relocation.BringIntoViewResponder
 import androidx.compose.foundation.relocation.bringIntoViewResponder
-import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.composed
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.geometry.Rect
 import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.unit.toSize
 
 @Suppress("IllegalExperimentalApiUsage") // TODO (b/233188423): Address before moving to beta
 @OptIn(ExperimentalFoundationApi::class)
-internal fun Modifier.bringIntoViewIfChildrenAreFocused(): Modifier = composed(
+internal fun Modifier.bringIntoViewIfChildrenAreFocused(
+    paddingValues: PaddingValues = PaddingValues()
+): Modifier = composed(
```

Comment:
Would be great if we can prevent using `composed` as this causes the whole modifier chain to be re-composed every time. I see the `bringIntoViewResponder` can't be delegated because it's internal, otherwise I'd suggest migrating to `Modifier.Node`.

Maybe at least add `// TODO`

Reply:
Thanks for catching this.  I added ToDo comment and filed #171 to track the issue.